test/e2e/kubectl/kubectl.go

```diff
@@ -1150,13 +1150,13 @@ metadata:
       "metadata": {
         %s
         "name": "subobject",
-        "namespace": "my-ns"
+        "namespace": %s
```

Comment:
Does namespace need to be quoted? 
```suggestion
        "namespace": "%s"
```